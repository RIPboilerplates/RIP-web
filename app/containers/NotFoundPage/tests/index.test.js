import React from 'react'
import renderer from 'utils/renderIntl'

import Component from '../'

describe('NotFoundPage Component', () => {
  describe('Renders', () => {
    it('default', () => {
      const component = renderer(<Component />).toJSON()
      expect(component).toMatchSnapshot()
    })
  })
})
