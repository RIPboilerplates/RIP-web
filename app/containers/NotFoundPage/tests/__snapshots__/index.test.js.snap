// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component Snapshot Renders default 1`] = `
<h1>
  <span>
    This is NotFoundPage component!
  </span>
</h1>
`;
